client/sites-dashboard/controller.tsx

```diff
@@ -68,6 +68,10 @@ export function sitesDashboard( context: PageJSContext, next: () => void ) {
 			<PageViewTracker path="/sites" title="Sites Management Page" delay={ 500 } />
 			<SitesDashboard
 				queryParams={ {
+					page: context.query.page ? parseInt( context.query.page ) : undefined,
+					perPage: context.query[ 'per-page' ]
```

Comment:
Do we want to allow users to set it to any value? Or should we accept the value from the parameter only if it's lower or equal to 96?

Reply:
> Do we want to allow users to set it to any value? Or should we accept the value from the parameter only if it's lower or equal to 96?

Given the value only modifies displayed state, and isn't passed through to an API call, I don't think we need to do any more processing than we already are.